### ISSUE TITLE
Bug typo in emotion-recognition-retail-0003 website while installing it.

### ISSUE THREAD

**Issue by razzOn2calculus** [opencv/open_model_zoo#795]
I found a typo when installing pretrained model in openvino emotion-recognition pretrained model . 
**[emotion-recognition](https://docs.openvinotoolkit.org/2019_R1/_emotions_recognition_retail_0003_description_emotions_recognition_retail_0003.html)**

I was getting issue when downloading pretrained model using command on openvino directory
```
python downloader.py --name emotion-recognition-retail-0003 --output_dir /home/workspace

No matching models: "emotion-recognition-retail-0003"
```
As documentation says, I had used that emotion-recognition-retail-0003  name from above link

But problem solved when i changed emotion to emotions
`python downloader.py --name emotions-recognition-retail-0003 --output_dir /home/workspace
`

**Comment by eaidova** [comment]
@razzOn2calculus thank you for your report, 
we will update emotions-recognition-retail-0003 docs.

**Comment by eaidova** [comment]
Fixed with docs for OpenVINO 2020.1
https://docs.openvinotoolkit.org/latest/_models_intel_emotions_recognition_retail_0003_description_emotions_recognition_retail_0003.html
